### DISCUSSION TITLE
How do you tell when data goes out of cache?

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [TabularEditor/TabularEditor3#1248] [Data Modeling]
Say i have 10 large models all built in SSAS (no other models)
1 model i decide to make no refreshes on for a week.
Say i have bookmarks in PBI that dictates what is ran (no one will do extra filtering or adhoc)
Say I go through 10 book marks. Each takes 10 seconds to display results initially.
When i go back through result page return immediately.
Shouldn't we expect that for each user for each day  the data would return immediately UNTIL there is new refresh?

I am not seeing that as the case. I thought i would be all in cache.
I assume something must happen that evicts it from cache
What tools do we have to see why it isn't in cache?
I guess what i am looking for is a tool that can show me what causes the eviction.
I don't think TE or DAX Studio can show me that.

There must be other processes from other models  that are happening that are affecting this.
Ideas?

**Comment by otykier** [comment]
Since this is related to the Power BI Service internals, you’re probably much better off asking this question on the fabric community forum: https://community.fabric.microsoft.com/t5/Power-BI-forums/ct-p/powerbi but I’ll leave this open, in case anyone here knows the answer.

**Reply by TinyTimothy2** [reply]
Thanks.  Since we use SSAS and Power BI RS, i thought this is actually on prem SSAS and server issue.  Even though are ON Prem PB RS has semantic models, I thought the SSAS server is treated as separated entity.
I thought maybe your tool (or dax studio) would have a hook to see.
I will check out the Fabric community.
